### DISCUSSION TITLE
Tabs

### DISCUSSION THREAD

**Discussion by LighthammerXP** [FreeTubeApp/FreeTube#4794] [General]
Greetings, 

I was REALLY hoping this would be a feature seen sooner then later, but it seems I need to ask!

Is there any plans/intents to add tabs to Freetube?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi there is an open issue for this: https://github.com/FreeTubeApp/FreeTube/issues/333

You will probably notice that the issue has been opened a long time ago but the thing is that there are way too many other things on our priority list than this issue. We are a small team of volunteers that work in our free time. So stuff like this wont get done by us until the higher priory stuff is addressed. We do take PR's from contributors outside our team, so feel free to address this if you want :)
